### PR TITLE
Dynalab lambda models

### PR DESCRIPTION
**Pull request by Ciroye**

### CODE REVIEW

File: new_builder_evaluation/app/api/endpoints/aimodels.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Copyright (c) MLCommons and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from fastapi import APIRouter
+
+from app.domain.aimodels import AIModels
```
Comment (remg1997) [inline new_builder_evaluation/app/api/endpoints/aimodels.py:7]: I like the implementation, but I'm not sure about the class name. AIModels seems to broad.

File: new_builder_evaluation/app/domain/aimodels.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        input_data = {}
+        for input in inputs:
+            name_input = input["name"]
+            input_data[name_input] = "test"
+        input_data["context"] = "test"
+        return input_data
+
+    def initiate_lambda_models(self):
+        models = self._model_repository.get_lambda_models()
+        while True:
```
Comment (remg1997) [inline new_builder_evaluation/app/domain/aimodels.py:30]: We should look for a better alternative to this loop.

**Review by remg1997 — APPROVED** [review]

File: new_builder_evaluation/app/api/endpoints/aimodels.py (hunk 1 above)
Comment (Ciroye) [reply new_builder_evaluation/app/api/endpoints/aimodels.py:7]: Any suggestions? I faced the same problem when I tried to set a name

File: new_builder_evaluation/app/domain/aimodels.py (hunk 2 above)
Comment (Ciroye) [reply new_builder_evaluation/app/domain/aimodels.py:30]: For now I think this is the only way but the good news is that the AWS team is working on a way to solve this kind of problem.

**Pull request merged**